### ISSUE TITLE
Cmake build 

### ISSUE THREAD

**Issue by whit2333** [JeffersonLab/analyzer#165]
```CMake Error at /usr/share/cmake-3.9/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find EVIO (missing: EVIO_LIBRARY EVIO_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.9/Modules/FindPackageHandleStandardArgs.cmake:377 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindEVIO.cmake:48 (find_package_handle_standard_args)
  hana_decode/CMakeLists.txt:15 (find_package)

```

Adding CONFIG in `hana_decode/CMakeLists.txt`:
```
find_package(EVIO REQUIRED CONFIG)
```
fixes it.

**Comment by hansenjo** [comment]
Commit d9d8001 adds a configuration option EVIO_HAS_CMAKE_CONFIG (default OFF) which, when enabled, runs find_package(EVIO) with the CONFIG option. Let me know if that works for you.

**Comment by whit2333** [comment]
Here without any manual intervention:
https://github.com/JeffersonLab/analyzer/pull/167

**Comment by hansenjo** [comment]
Merged #167 in commit 83ee64e
